src/sidebar/components/Annotation/AnnotationBody.js

```diff
@@ -18,6 +18,34 @@ import TagListItem from '../TagListItem';
  * @typedef {import("../../../types/config").SidebarSettings} SidebarSettings
  */
 
+/**
+ * Button to expand or collapse the annotation excerpt (content)
+ * @param {object} props
+ *   @param {string} [props.classes]
+ *   @param {(collapse:boolean) => void} props.setIsCollapsed
+ *   @param {boolean} props.isCollapsed
+ */
+function ToggleExcerptButton({ classes, setIsCollapsed, isCollapsed }) {
+  const toggleText = isCollapsed ? 'More' : 'Less';
+  return (
+    <LabeledButton
+      classes={classnames('text-grey-7 font-normal', classes)}
+      expanded={!isCollapsed}
+      onClick={() => setIsCollapsed(!isCollapsed)}
+      title={`Toggle visibility of full annotation text: Show ${toggleText}`}
+    >
+      <div className="flex items-center gap-x-2">
+        <Icon
+          classes="!text-tiny"
```

Comment:
This rule isn't new but I wanted to call it out as a smell that arises because we need to refactor the implementation of `Icon`; it violates specificity conventions (that were, to be fair, defined after its creation). It should be possible to use a TW utility class on a component; there's an over-specific rule on `Icon` that makes it necessary to use !important. FOR NOW.

Reply:
Worth noting this in an inline comment?

Reply:
Will do, and also, amendment, it's the `LabeledButton` component at fault here, not `Icon`. `LabeledButton` asserts some styling on any contained `svg` that has high specificity.